src/why3prop.ml

```diff
@@ -0,0 +1,61 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Why3
+
+(* a type to present the list of why3 constants and lambdapi terms *)
+type cnst_table = (term * Term.lsymbol) list
```

Comment:
What is Term.lsymbol? Where is it defined?

Reply:
Ah, it's a Why3 module I guess. It would be good to prefix it by Why3 to avoid confusions (and use "open Why3" only locally?).

Reply:
Indeed, avoid `open Why3`.